### PR TITLE
[Update] Modularize display-information category

### PR DESCRIPTION
**Pull request by Luiy0**
### Description

- Updated, modularized, and removed deprecated code (colors) from the samples in the `display-information` category.
- `sketch-on-map` sample is failing due to an invalid URL or resource not found and is likely FXML related.
- `display-annotation` and `display-dimensions` samples do not have a `gradle-wrapper.properties` file.

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by Luiy0** [comment]
Assigning @jenmerritt for a first review please.

File: display_information/control-annotation-sublayer-visibility/src/main/java/com/esri/samples/control_annotation_sublayer_visibility/ControlAnnotationSublayerVisibilitySample.java [hunk 1]
```diff
@@ -28,7 +28,6 @@
 import javafx.scene.layout.CornerRadii;
 import javafx.scene.layout.StackPane;
 import javafx.scene.layout.VBox;
-import javafx.scene.paint.Color;
```
Comment (jenmerritt) [inline display_information/control-annotation-sublayer-visibility/src/main/java/com/esri/samples/control_annotation_sublayer_visibility/ControlAnnotationSublayerVisibilitySample.java:31]: Was this just a removed unused import?

File: display_information/sketch-on-map/src/main/resources/sketch_on_map/main.fxml [hunk 2]
```diff
@@ -25,7 +25,7 @@
 <?import javafx.scene.image.Image?>
```
Comment (jenmerritt) [inline display_information/sketch-on-map/src/main/resources/sketch_on_map/main.fxml:25]: I seem to be getting an exception when I run this sample, can you check

**Review by jenmerritt — COMMENTED** [review]
Changes look good but there's one sample with an run error if you could check that out
@Luiy0

File: display_information/control-annotation-sublayer-visibility/src/main/java/com/esri/samples/control_annotation_sublayer_visibility/ControlAnnotationSublayerVisibilitySample.java (hunk 1 above)
Comment (Luiy0) [reply display_information/control-annotation-sublayer-visibility/src/main/java/com/esri/samples/control_annotation_sublayer_visibility/ControlAnnotationSublayerVisibilitySample.java:31]: Correct. I've double checked and `Color` is not being used anywhere.

File: display_information/sketch-on-map/src/main/resources/sketch_on_map/main.fxml (hunk 2 above)
Comment (Luiy0) [reply display_information/sketch-on-map/src/main/resources/sketch_on_map/main.fxml:25]: Found a quick fix, pushing the commit.

**Review by jenmerritt — APPROVED** [review]
LGTM @Luiy0 this can be merged since it's just modularization

**Pull request merged**